SCClassLibrary/Common/Quarks/Quarks.sc

```diff
@@ -203,22 +204,31 @@ Quarks {
 		});
 
 		"Installing %".format(quark.name).postln;
-
+		// add currently installing quark to 'installing' Array
+		installing = (installing.size == 0).if{ quark.bubble }{ installing ++ quark.bubble };
 		quark.checkout();
-		if(quark.isCompatible().not, {
+		quark.isCompatible().not.if{
```

Comment:
I notice this method mixes control flow syntaxes, did you go with this way because it was the "nearest" style?
Better to just convert them to the [recommended](https://github.com/supercollider/supercollider/wiki/Style-Guidelines%3A-SuperCollider#recommendation-use-trailing-closure-syntax-whenever-possible) trailing closure style?

Reply:
I went with `(0==1).if{ true } { false}` throughout - I didn't take the style guideline as insisting on `if(0==1){ true }{ false }` since both forms are trailing closures but I'm happy to change all if you think its best.

Reply:
OK I went ahead and changed all